packages/taquito-rpc/test/data/rpc-responses.ts

```diff
@@ -5769,3 +5769,410 @@ export const smartRollupExecuteOutboxMessageResponse = {
     ],
   ],
 };
+
+export const smartRollupCementSample = {
```

Comment:
could you rename these to `smartRollupCementResponse` like the other examples for consistency? 

same with the other additional smart rollup operations